casper/src/main/scala/io/casperlabs/casper/equivocations/EquivocationTracker.scala

```diff
@@ -0,0 +1,33 @@
+package io.casperlabs.casper.equivocations
+
+import io.casperlabs.casper.Estimator.Validator
+
+// Stores the lowest rank of any base block, that is, a block from the equivocating validator which
+// precedes the two or more blocks which equivocated by sharing the same sequence number.
+class EquivocationTracker(private val map: Map[Validator, Long]) {
```

Comment:
Nitpick but it should be plural IMO.
```suggestion
class EquivocationsTracker(private val map: Map[Validator, Long]) {
```

Reply:
Done.